tests/wp-cldr-tests.php

```diff
@@ -30,18 +30,31 @@ public function test_territory_name() {
 		// Test region names.
 		$this->assertEquals( 'Afrique', $this->cldr->territory_name( '002', 'fr_FR' ) );
 		$this->assertEquals( '亚洲', $this->cldr->territory_name( '142', 'zh-cn' ) );
+
+		// Test some bad slugs.
```

Comment:
yep. the tests _should_ all just return a blank string.  tests are just checking to be sure we get that expected result.

this came up when some code coverage reports showed we weren't testing the handling of `null` / empty strings.